index.js

```diff
@@ -46,16 +38,15 @@ cmd.rm = (id, opts) => {
 }
 
 cmd.update = (id, endpoint, secret, opts) => {
-  opts = HooksConfig(opts)
   validate('SSSO', [id, endpoint, secret, opts])
-  return fetch.json(`/-/npm/v1/hooks/hook/${eu(id)}`, opts.concat({
+  return fetch.json(`/-/npm/v1/hooks/hook/${eu(id)}`, {
+    ...opts,
     method: 'PUT',
     body: {endpoint, secret}
-  }, opts))
+  })
 }
 
 cmd.find = (id, opts) => {
```

Comment:
Should get a default value in all of these, since they might not get an options object.  `cmd.update = (..., opts = {}) => {`  (Previously, the HooksConfig function would turn `undefined` into an object.)